pkg/utils/json/json.go

```diff
@@ -79,12 +44,3 @@ func removeListFields(config, live []interface{}) []interface{} {
 	}
 	return result
 }
-
-// MustMarshal is a convenience function to marshal an object successfully or panic
-func MustMarshal(v interface{}) []byte {
```

Comment:
I think this method should have stayed here.

Reply:
I removed it because it is not used. I personally see gitops-engine as a library and as such I think it'd be best for it to have a small clean API with as little extra stuff exposed as possible. Reducing the exposed API surface has the following benefits:
- It gives maintainers the biggest flexibility to change how things work without breaking users. If a method/function/type is not exposed then changing it will not break any users.
- It makes the library easy to use.

Reply:
I agree, the less methods we expose through the API, the better. The JSON utils were probably never supposed to be consumed by consumers anyway, so better to break existing consumers now while we're still pre-v1.0 than later. Thanks for your response. :+1: